metricproviders/prometheus/prometheus.go

```diff
@@ -22,6 +26,7 @@ const (
 	// ResolvedPrometheusQuery is used as the key for storing the resolved prometheus query in the metrics result
 	// metadata object.
 	ResolvedPrometheusQuery = "ResolvedPrometheusQuery"
+	Address                 = "PROMETHEUS_ADDRESS"
```

Comment:
1. `PROMETHEUS_ADDRESS` could use a better configuration convention. I think we can go with `ARGO_ROLLOUTS_PROMETHEUS_ADDRESS`. In the future, we will have more of such environment variables for configuration and the `ARGO_ROLLOUTS` is a prefix to group rollouts specific env vars together. We do the same with `ARGOCD_`

2. nit: The global variable name `Address` is not very specific. Commonly if something is used as an env var, you can prefix it with `EnvVar<keyname>`. e.g. `EnvVarArgoRolloutsPrometheusAddress`